WordPress/Classes/Services/ReaderPostService.m

```diff
@@ -569,11 +613,11 @@ - (void)removeGapMarkerForTopic:(ReaderAbstractTopic *)topic ifNewPostsOverlapMa
 {
     ReaderGapMarker *gapMarker = [self gapMarkerForTopic:topic];
     if (gapMarker) {
-        NSDate *newestPostDate = ((ReaderPost *)newPosts.firstObject).sortDate;
-        NSDate *oldestPostDate = ((ReaderPost *)newPosts.lastObject).sortDate;
-        NSDate *gapDate = gapMarker.sortDate;
+        CGFloat highestRank = [((ReaderPost *)newPosts.firstObject).sortRank doubleValue];
+        CGFloat lowestRank = [((ReaderPost *)newPosts.lastObject).sortRank doubleValue];
+        CGFloat gapRank = [gapMarker.sortRank doubleValue];
```

Comment:
(Maybe) those three should be a `double`? (since `doubleValue` actually returns *double*)